Lib/tomllib/_parser.py

```diff
@@ -214,7 +214,7 @@ class Flags:
     EXPLICIT_NEST = 1
 
     def __init__(self) -> None:
-        self._flags: dict[str, dict] = {}
+        self._flags: dict[str, dict[Any, Any]] = {}
```

Comment:
> It is fully typed already, we just need to run the CI if something changes.

Seems it wasn't fully typed already?

Please can you also submit these changes to https://github.com/hukkin/tomli ?

Reply:
Should I do it before or after this PR?

Reply:
I'd suggest there first, because we're sort of downstream, but no strong preference from me.

Let's ask @hukkin, @encukou and @hauntsaninja.